desktop/src/main/java/bisq/desktop/main/MainViewModel.java

```diff
@@ -441,10 +451,12 @@ private void setupBtcNumPeersWatcher() {
                 checkNumberOfBtcPeersTimer = UserThread.runAfter(() -> {
                     // check again numPeers
                     if (walletsSetup.numPeersProperty().get() == 0) {
-                        if (localBitcoinNode.isDetected())
-                            getWalletServiceErrorMsg().set(Res.get("mainView.networkWarning.localhostBitcoinLost", Res.getBaseCurrencyName().toLowerCase()));
```

Comment:
Apart from the `.isDetected()` -> `.isUsable.get()` line, these are just formatting changes.